m4/ax_python_module.m4

```diff
@@ -21,13 +23,13 @@
 #   and this notice are preserved. This file is offered as-is, without any
 #   warranty.
 
-#serial 6
+#serial 7
 
 AU_ALIAS([AC_PYTHON_MODULE], [AX_PYTHON_MODULE])
 AC_DEFUN([AX_PYTHON_MODULE],[
     if test -z $PYTHON;
     then
-        PYTHON="python"
+        PYTHON="$3"
```

Comment:
Currently that will confuse this code into thinking that the specified Python module can't be found. Would you prefer that we give a more helpful error message in this case saying too few parameters were passed? Alternatively, we could default the third parameter to "python."